src/openrct2/game.c

```diff
@@ -317,6 +317,25 @@ void game_update()
         network_process_game_commands();
     }
 
+    // Update the game one or more times
+    for (i = 0; i < numUpdates; i++) {
```

Comment:
doesn't look like loop counter is being used outside of the loop. Can you move variable declaration into the loop? `for (int i…`